py_mmd_tools/nc_to_mmd.py

```diff
@@ -262,8 +273,33 @@ def __init__(
             )
         super(Nc_to_mmd, self).__init__()
         self.output_file = output_file
+
         if not json_input:
             self.netcdf_file = os.path.abspath(netcdf_file)
+            self.target_nc_filename = self.netcdf_file
+            self.ncin = self.read_nc_file(self.netcdf_file)
+            self.check_attributes_not_empty(self.ncin)
+            self.file_size = pathlib.Path(self.netcdf_file).stat().st_size / (1024 * 1024)
```

Comment:
This calculation is in conflict with docstring line 260. Bytes to GB is 1/1024**3

Reply:
set to MB in the docstring